src/components/containers/WorkflowContainer.tsx

```diff
@@ -257,7 +256,7 @@ const WorkflowPane = (props: ElementProps) => {
         }
       }}
     >
-      <PreviewToolbox />
+      {/* <PreviewToolbox /> */}
```

Comment:
Intentional?

Reply:
Yep! James is gonna rework the design. Hiding it for now.